test/64.js

```diff
@@ -9,4 +9,8 @@ describe('reverse word order in a sentence', () => {
   it('should return I', () => {
     expect(solution('I')).eql('I');
   });
+  it.only('should reverse word order', () => {
```

Comment:
maybe this will help! https://github.com/llipio/algorithms/wiki/Git-and-branching